### DISCUSSION TITLE
Trigger on push events to pull requests?

### DISCUSSION THREAD

**Discussion by jeff-hernandez** [community/community#26366] [Actions]
<p>On push trigger events, the workflow runs on all pushes to a repository branch. Is it possible to run on all pushes only if the repository branch is an opened pull request?</p>
<pre><code class="lang-yaml">on:
  push:
    types: [pull_request]

...
</code></pre>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/jeff-hernandez">@jeff-hernandez</a>, welcome to the GitHub Support Community! Workflows can only be triggered by the events listed on this page:</p>
<p><a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows">https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows</a></p>
<p><code>pull_request</code> isn’t a valid type of <code>push</code> event so this syntax won’t work. What you’re likely looking for is the <code>synchronize</code> type of the <code>pull_request</code> event. This will trigger your workflow each time the HEAD branch is updated in a pull request—i.e. when it’s pushed to.</p>
<p><a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request">https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#pull_request</a></p>
<pre><code class="lang-yaml">on:
  pull_request:
    types: [synchronize]
</code></pre>
